pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -83,7 +84,7 @@ def default_sampler_config(self) -> Dict:
     @property
     def output_var(self):
         """Defines target variable for the model"""
-        return "y"
+        return "likelihood"
```

Comment:
I think y is a better good name. likelihood is about the kind of variable it is. That would be like renaming betas to "prior"

Reply:
I change in the model definition then!